### DISCUSSION TITLE
New to Open-Source Contribution

### DISCUSSION THREAD

**Discussion by AyushBarai** [microsoft/AirSim#4878] [General]
Hi Everyone,
I recently started contributing to open-source projects, and I'm very eager to begin contributing to AirSim. Any guidance or assistance would be greatly appreciated as I embark on this new journey. Thank you in advance for your support and insights!

**Comment by EXPX3** [comment]
HI @AyushBarai 
were you able to follow the doccumentation and do the basic airsim setup? here is the doccumentation [link](https://microsoft.github.io/AirSim/).
would suggest you start going through the doccumenattion and run some basic examples. and then going through the Issues, closed/open PullRequests etc you can dig deeper. Once you gain your base at some point you will find a way to contribute here. I wish you all the best! and welcome! :)
